src/components/collection_voting_slab/MultiDatasetVoteSlab.js

```diff
@@ -79,17 +79,18 @@ export default function MultiDatasetVoteSlab(props) {
       h(Alert, {
         title: 'Voting is disabled since not all elections are open.',
         type: 'danger',
-        isRendered: !allOpenElections
+        isRendered: !adminPage && !allOpenElections
       }),
       h(CollectionSubmitVoteBox, {
         question: 'Should data access be granted to this applicant?',
         votes: currentUserVotes,
         isFinal: isChair,
-        isDisabled: readOnly || isEmpty(currentUserVotes) || !allOpenElections,
+        isDisabled: adminPage || readOnly || isEmpty(currentUserVotes) || !allOpenElections,
```

Comment:
I think the ticket mentions not showing the vote buttons/rationale textbox at all for the admin page. I can see the use of showing them in the case that the admin is also a chair that voted on this collection, but I'm wondering if it would make more sense to just not show the buttons since that is not the majority of cases

Reply:
1. Currently the review page shows the final vote through the buttons (highlighted background color, white if vote not submitted), the rationale through the textfield, and the member votes through the table. The admin should be able to see everything and I think the current template does that very well. 

Removing the buttons and rationale just takes information away, so it's just easier to make everything read-only and to ensure that no data filtering processes are fired for the admin view.